src/frontend/screens/MapScreen/track/UserTooltipMarker.tsx

```diff
@@ -3,11 +3,11 @@ import {StyleSheet, Text, View} from 'react-native';
 import React from 'react';
 import {usePersistedTrack} from '../../../hooks/persistedState/usePersistedTrack';
 import {useTrackTimerContext} from '../../../contexts/TrackTimerContext';
-import {useSharedLocationContext} from '../../../contexts/SharedLocationContext';
+import {useLocation} from '../../../hooks/useLocation';
 
 export const UserTooltipMarker = () => {
   const {timer} = useTrackTimerContext();
-  const {locationState} = useSharedLocationContext();
+  const locationState = useLocation();
```

Comment:
This needs a filter to avoid too frequent map-redraws.

Reply:
https://github.com/digidem/comapeo-mobile/pull/387#discussion_r1630203254